joy2twist/config/joy2twist.yaml

```diff
@@ -0,0 +1,22 @@
+/**:
```

Comment:
If I understand correctly it sets parameters for all nodes, in this case it shouldn't matter, but is it a good practice to use it?

Reply:
Params are automatically under "node_name" namespace

Reply:
understood, my bad, it's alright